commitlintplugins.test.ts

```diff
@@ -219,3 +219,26 @@ test('type-space-after-comma2', () => {
     expect(typeSpaceAfterComma2.status).toBe(0);
 });
 
+test('wip-with-number-1', () => {
+    let commitMsgWithNoNumberAfterWIP = "WIP: bla bla blah";
```

Comment:
@parhamsaremi the most basic test should be the simplest test you can think of that can break it (remember TDD philosophy?), and this test is not simple; the commit message should just be "WIP"

Reply:
hmmm right. I though the most basic one is a WIP with message but you are right a simple "WIP" is the easiest test. I'm working on it.